docs/spec/spec.md

```diff
@@ -8,7 +8,7 @@ _Edition_: Swan Lake
 
 ## Introduction
 
-This is the specification for the Ballerina persist standard library of [Ballerina language](https://ballerina.io/), which provides functionality to store and query data easily.  
+This is the specification for the Ballerina persist standard library of [Ballerina language](https://ballerina.io/), which provides functionality to store and query data easily.
```

Comment:
```suggestion
This is the specification for the `persist` standard library of the [Ballerina language](https://ballerina.io/), which provides functionality to store and query data easily.
```

Reply:
Fixed in [e4dc132](https://github.com/ballerina-platform/module-ballerina-persist/pull/109/commits/e4dc1324152adf53e52a1ca4dea90ffacb44ae7a)